clarity_ext/context.py

```diff
@@ -9,9 +7,10 @@
 from clarity_ext.repository import StepRepository
 from clarity_ext import utils
 from clarity_ext.driverfile import OSService
+from clarity_ext.domain.udf import Udf
 
 
-class ExtensionContext(object):
+class ExtensionContext(Udf):
```

Comment:
One might argue that this could be renamed for clarification (in a future commit), e.g. `SupportsUdfsMixin`.